common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -198,7 +198,7 @@ public class RssClientConf {
   public static final ConfigOption<Boolean> RSS_CLIENT_REMOTE_STORAGE_USE_LOCAL_CONF_ENABLED =
       ConfigOptions.key("rss.client.remote.storage.useLocalConfAsDefault")
           .booleanType()
-          .defaultValue(false)
+          .defaultValue(true)
```

Comment:
Why do we need to change the default value?

Reply:
I hope so, that will make shuffle server without hadoop env by default